backend/src/main/java/com/woowacourse/kkogkkog/domain/BaseEntity.java

```diff
@@ -0,0 +1,17 @@
+package com.woowacourse.kkogkkog.domain;
+
+import java.time.LocalDateTime;
+import javax.persistence.EntityListeners;
+import javax.persistence.MappedSuperclass;
+import lombok.Getter;
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+@EntityListeners(AuditingEntityListener.class)
+@MappedSuperclass
+@Getter
+public abstract class BaseEntity {
+
+    @CreatedDate
+    private LocalDateTime createdAt;
+}
```

Comment:
createdAt 상속받을 수 있도록 protected 필드로 수정하고, 자식클래스에 어차피 @Getter 있으니 제외해도 되지 않나 싶어요.

```suggestion
@EntityListeners(AuditingEntityListener.class)
@MappedSuperclass
public abstract class BaseEntity {

    @CreatedDate
    @Column(nullable = false)
    protected LocalDateTime createdAt;
}
```

Reply:
이렇게 하니 `MemberHistory`에서 `getter` 사용이 불가능하네요 ㅠㅠ